IntegrationTests/ContentBlockingRulesTests.swift

```diff
@@ -50,10 +50,8 @@ class ContentBlockingRulesTests: XCTestCase {
 
 extension Array where Element == ContentBlockerRule {
     func findExactFilter(filter: String) -> ContentBlockerRule? {
-        for rule in self {
-            if rule.trigger.urlFilter == filter {
-                return rule
-            }
+        for rule in self where rule.trigger.urlFilter == filter {
```

Comment:
forgot about this syntax 👍

Reply:
To be honest I did too! But SwiftLint actually detected a single `if` statement inside the body of the loop and recommended this, pretty impressive.